files/es/web/api/element/closest/index.html

```diff
@@ -34,7 +34,7 @@ <h3 id="Valor_del_resultado">Valor del resultado</h3>
 <h3 id="Excepciones">Excepciones</h3>
 
 <ul>
- <li>Se lanza una excepción {{exception("SyntaxError")}} si <em><code>selectors</code></em> no es una lista de selectores válida.</li>
+  <li>Se lanza una excepción <a href="/es/docs/Web/API/DOMException#syntaxerror">`SyntaxError`</a> si <em><code>selectors</code></em> no es una lista de selectores válida.</li>
```

Comment:
La costumbre del `md` XD

```suggestion
  <li>Se lanza una excepción <a href="/es/docs/Web/API/DOMException#syntaxerror"><code>SyntaxError</code></a> si <em><code>selectors</code></em> no es una lista de selectores válida.</li>
```